javascripts/discourse-linkify/initializers/initialize-discourse-linkify.js.es6

```diff
@@ -40,12 +49,12 @@ export default {
       actions.forEach(readInputList);
         
       api.decorateCooked($elem => {
-        actions.forEach(a => {
-          if (Object.keys(a.inputs).length > 0) {
-            modifyNode($elem[0], a, skipTags)
+        actions.forEach(action => {
+          if (Object.keys(action.inputs).length > 0) {
+            traverseNodes($elem[0], action, skipTags, skipClasses)
           }
         });
-      });
+      }, {'id': 'linkify-words-theme'});
```

Comment:
I added an 'id' to the api.decorateCooked call as per

javascripts/discourse-linkify/initializers/initialize-discourse-linkify.js.es6

Is that okay?

Reply:
Yup, and `id` is always a good idea 👍